decidim-bulletin-board-app/app/commands/process_key_ceremony_step.rb

```diff
@@ -32,6 +32,7 @@ def call
 
       election.log_entries << log_entry
       log_entry.save!
+      LogEntryNotifier.new(log_entry).notify_subscribers
```

Comment:
❤️ this!
Should we do this as the last step, to avoid notifying users after rollbacking the transaction. I know it is not big deal, but I like to let these kind of steps for the end of the transaction.

Reply:
Oh totally! I forgot about the transaction. I'll move it outside of it!